### PR TITLE
source: add print-sample-config flag for operate-source

### LINKED ISSUE #782: give a example of file in operate-source help
give a example of file in operate-source help

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What problem does this PR solve? 
close #782 

### What is changed and how it works?
- add print-sample-config flag for operate-source

Tests 
 - Manual test (add detailed scripts or steps below)

### CODE REVIEW

File: dm/ctl/master/operate_source.go [hunk 1]
```diff
@@ -48,6 +53,27 @@ func convertCmdType(t string) pb.SourceOp {
 
 // operateMysqlFunc does migrate relay request
 func operateSourceFunc(cmd *cobra.Command, _ []string) {
+
```
Comment (WangXiangUSTC) [inline dm/ctl/master/operate_source.go:56]: how about removing this line

**Comment by lance6716** [comment]
> @lance6716 you may need to update to YAML in #828 after this merged?

Yes, I'll add a todo in my code, can merge now

File: Makefile [hunk 2]
```diff
@@ -41,9 +41,11 @@ MAC       := "Darwin"
 ifeq ($(ARCH), $(LINUX))
 	LDFLAGS += -X "github.com/pingcap/dm/dm/worker.SampleConfigFile=$(shell cat dm/worker/dm-worker.toml | base64 -w 0)"
 	LDFLAGS += -X "github.com/pingcap/dm/dm/master.SampleConfigFile=$(shell cat dm/master/dm-master.toml | base64 -w 0)"
+	LDFLAGS += -X "github.com/pingcap/dm/dm/config.SampleConfigFile=$(shell cat dm/master/source.toml | base64 -w 0)"
```
Comment (lance6716) [inline Makefile:44]: wow go could do these things

**Pull request merged**